Updates to Player Guide Brief, Player Guide Full, and start of GM Guide Brief (#183)
DeckofAdventures/TheGame: docs/src/1_Mechanics/01_PlayerGuide_Full.md

  drawing from their respective decks to inform the outcome.

2. Jokers and Aces are Fate Cards and should be reserved in one's hand, for both players
  and the GM. All other cards go directly into a Discard Pile.

3. Players will use their deck to do most anything; it represents their character's
  stamina until they complete a Rest. When the Action Deck is empty and a player has to
  shuffle between rests, they suffer from one level of
  [Fatigue](./01_PlayerGuide_Full.md#status-conditions) until they would shuffle after a
  Full Rest.

## Fate Cards

Fate Cards (Jokers and Aces) are a chance to control your fate. They permit the player
to choose when they want a second chance at a key moment.

1. After a Full Rest, both Jokers return to the player's hand as Fate Cards, if they're
not there already.

2. Aces drawn from the deck are played as usual, but remain in the player's hand as a
Fate Card rather than going to the Discard Pile. Aces reserved as Fate Cards are

> reshuffled back into the deck after a Full Rest.

Author: I believe we had discussed Full Rests resetting everything except for Joker Fate Cards, which would indicate that Aces do get reshuffled at the end of a Full Rest (makes it easier to 'reset' after a Full Rest as well, where a player doesn't need to remember how many Aces and which Aces are to remain in their hand). 

If Aces remain in hand, and a Player holds onto them between Rests, then a Player may conceivably start a new day at 48 stamina with no aces in their deck and 6 fate cards at the start of their day, which may result in players hoarding the cards and make it more difficult for a GM to balance.

    1. *Push Fate*: redraw any check.

>     2. *Exchange Fate*: gain Power Points equal to the number of Fate Cards used, up to
>        your maximum Power Points value.

Author: ```suggestion
    2. *Exchange Fate*: gain Power Points equal to the number of Fate Cards used.
```

Difficulty Range (DR) is the range around the TC the player might hope to draw, like
the circles around the bullseye.

The Difficulty Range is decided by the GM and they can choose whether or not to reveal
it before a check is made. Players may consult with their GM to determine if another
Attribute or Skill may be used instead. Once the check is called for and the relevant
skill is selected, players must make the check. A player will draw one or more cards.
Immediately after drawing, a player will apply any relevant Modifiers before reporting
the outcome and whether or not they have successfully fallen within the DR. A Modifier
is added to the DR when a check is made, effectively widening the DR and making it
easier to succeed. A Modifier is indicated on the character sheet next to the relevant
Skill or Attribute related to the Check. Think of it as a character's ability to 'aim' a
check to get closer to the TC.

The GM may describe the nature of the outcome or announce the degree of success. This
may be simply to report the success or failure of an action, or provide a more detailed
description of the outcome. The GM may ask the player to describe how their character

> successfully accomplishes the action depending on the scenario.

Author: I had edited out the `this is the character's ability to 'aim' a check to get closer to the TC` since we decided that the Mod should widen the DR and make the Effective DR a higher number. So it's less about Aiming and more about widening the DR to make it easier to succeed. I'll revisit this code suggestion to see how we resolve both, since I see widening the DR is included towards the bottom of the first pgf.

drawing before N cards are drawn.

2. Having the Lower Hand (N) means you draw N additional cards and select the card that
is furthest from the TC after applying all relevant mechanics. A player must draw all N
cards.

In these cases, all cards drawn are discarded. If N is not specified, draw 1 extra.
Multiple Upper/Lower hand effects can compound or even out. If a rule specifies
Upper/Lower Hand (N+), the effect can apply multiple times. N cards are drawn, +1 for

> each instance the effect is applied. For example, running through your Action Deck

Author: ```suggestion
each instance the effect is applied. For example, running through your deck
```

> each instance the effect is applied. For example, running through your Action Deck

Author: Agreed - I think `Action Deck` may be included elsewhere from an earlier version we can go through and make sure it's simply replaced with `deck` like I'm doing below.

When a character attempts to resist or shrug off a negative effect, they make a Save.
This could be caused by a Power, Status Condition, or a environmental effect. When a
character makes a Save, they draw a Check against their own TC and apply their relevant
Attribute Modifier. The Attribute and DR are explicitly set by the initiating effect and
cannot be altered. If the effect forcing a Save is a Power, the DR is 3 minus half of
the initiator's relevant Skill modifier. Saves may still have consequences even on a
success, as specified by the initiating effect.

### In Combat

Some stories involve hostile forces that will threaten players with violence. If
violence is unavoidable, the GM decides when Combat begins and then a separate set of
rules take effect. Combat in Deck of Adventures is designed to be fast-paced, while
leaving a lot of room for player teamwork and creativity. To keep up the pace, players
should be sure to study their character sheets and know their options ahead of combat,
and use others’ turns to make their choices. Deck of Adventures is all about teamwork,

> so communication is encouraged.

Author: Not sure what you're referring to here.

> so communication is encouraged.

Reviewer: I had previously included a rule that would've granted bonuses to PCs for holding turns and then attacking together. It was too vulnerable to exploitation, so we removed it, removing the direct mechanical benefit for coordination

Characters do not discard their Target Cards after combat ends. This TC represents the
character until the start of the next combat.

If the Players are defeated in combat, the GM determines what happens next in the story.
If the scenario was of life or death importance, the Player Characters may be killed
outright. In less tense scenarios, the Player Characters may simply be left knocked out,
imprisoned, or some other relevant circumstance.

> #### Status Conditions

Reviewer: These were edited and then re-wrapped in the same commit, making it very difficult to see the changes

  make one Action on their turn, not both. Some Powers can cause the Stunned Condition.

  2. **Entangled** A character may expend a Major Action to make a DR 3 STR Save. Until they
  are successful, their Speed is 0 and any Agility Saves are made with Lower Hand.

  3. **Knocked Down** A character must expend a Minor Action to get up. Their movement speed
  is halved, rounding up. When Knocked Down, attackers within 1 space draw with Upper Hand
  and attackers further than 1 space away draw with Lower Hand.  All STR and AGL Saves are
  made with Lower Hand. A character can use 1 speed to willingly go prone as part of their
  movement, gaining the Knocked Down status.

  4. **Blinded, Deafened** At the start of their turn, as a Free Action, a character makes
  a DR 3 Intuition Save. If they fail, they draw Lower Hand (3) for any action requiring
  the relevant sense.

  5. **Knocked Out** Knocked Out (KO) characters are unable to act until treated by a Craft
  check or a Healing Power (DR 5 vs. the KO character's TC, minus 1 for each turn passed
  since the effect onset). If a Knocked Out Dealer is attacked further, the Player begins

>   an Epic Event alone to avoid *Death*.

Reviewer: In my initial draft, I made no mention of Death and kept it implied for a more family friendly vibe

  6. **Burned** As a Minor Action, a character can make a DR 3 GUT Save to stop the
  burning. On failure, they take 1 damage bypassing AP.

  7. **Poisoned N** Each poisoning effect adds +1 Poisoned. At the start of their turn,
  as a Free Action, a character makes a DR 3 VIT Save to shake off the effects of the
  poison. On failure, they discard N+1 cards.

>   8. **Frozen** Movement speed is reduced by half and all Strength and Agility based

Reviewer: shorthand consistecy
```suggestion
  8. **Frozen** Movement speed is reduced by half and all STR and AGL based
```

  Major Action available to them, as directed by the creature that caused the condition.
  Another creature within 6 spaces may use a Major action to end this effect. This
  effect ends automatically if the target is damaged by the creature that caused the
  condition.

  12. **Invisible** When making a Check to detect an Invisible creature, draw with Lower
  Hand. Unless otherwise stated, using a Power ends the Invisible condition. The attack
  that ends Invisibility is drawn at Upper Hand.

  13. **Grappled** Movement speed is reduced to 0 and a Grappled creature cannot gain
  the benefit of additional speed. They have Lower Hand when using any Power. A
  creature can end the Grappled condition by making a successful Contested Brute or
  Finesse Check against target's Brute.

  14. **Dead** If a Knocked Out Dealer is attacked further, they begin an Epic Event. On
  failing the Epic Event, the Dealer is dead in the game world. They live on in the
  Player's memory and the other characters of the world. Players are encouraged to
  create a new Dealer and work with their GM to introduce that character in a future

>   session. Some GMs may decide that Player Character *Death* is not part of their story.

Reviewer: This forces mechanics where I would rather DM choice. What about an adventure to retrieve a party member from the great beyond?

Author: Will remove for now, and potentially re-add as a section in the GM's guide for optional ways to handle Death.

gain the effects of cover.

**Partially Covered**: When a creature is still visible behind something but it is still
  reasonable to hit them. While partially covered the creature's AR is reduced by -1,
  making them harder to hit. For example...

- Standing directly between another creature and an attacker.
- Standing behind a crate or barrel that is half the creature's height.

**Fully Covered**: when a creature is nearly impossible to see behind something and it

>   is unlikely to land a hit on them. While fully covered, the creature's AR is reduced

Reviewer: What's the difference between character and creature? If relevant, please specify. It not, please revise for consistency

Author: We currently have 110 references to `character` throughout the PGF and 41 references to `creature`.

`creature` is mostly referenced when talking about mechanics like Status Conditions, Cover, Lighting, and Combat Rules. Character is used everywhere else. 

Should everything refer to `character` instead of `creature`? or do we indicate the difference in some way where a `character` is a humanoid with the ability to communicate whereas a `creature` is anything present in the game world? Not sure how we'd clarify the difference, if any, but open to feedback.

Reviewer: Sounds like the treading the same ground as `Dealer`, `Bystander`, `NPC`. You could pick `Character` as a hypernym and add it to the Glossary. Looks like NPC might need adding there too

  Experience Points to spend when they are added to the Character Sheet during
  Character Creation.

- Powers are special features, boons or strengths, that cost 1 or 2 Experience Points.
  If a Power has a Power Point cost, add the higher PP value to your total PP. Powers
  may have additional requirements such as Role or Level.

For a full set of character creation options, see our
[Character Creation page](./03_CharacterCreation.md).

### Leveling up

As your character journeys through the world and accumulates experience, they will
*Level Up*. Every character starts at Level 1 and gains experience over time.

When leveling up, characters:

> - Add 1 to their maximum HP

Reviewer: Is this the same as the point of HP that reflects level? If so, this wording implies an additional point.
```suggestion
```
